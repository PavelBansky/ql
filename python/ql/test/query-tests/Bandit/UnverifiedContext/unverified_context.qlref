Bandit/unverified_context.ql