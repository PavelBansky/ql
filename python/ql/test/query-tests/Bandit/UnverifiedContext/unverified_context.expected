| Test.py:7:11:7:40 | Attribute | Unverified context |
